### PR TITLE
feat(VirtualizedList): add a CellDatetimeRenderer

### PR DESCRIPTION
**Pull request by sgendre**
**What is the problem this PR is trying to solve?**
DateTime are transformed in text too early, meaning that sorting will deal with computed lexicographic value which are wrong for comparaison.

**What is the chosen solution to this problem?**
Keep initial timestamp and provide a specific renderer that can be parameterized from within settings.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/VirtualizedList/CellDatetime/CellDatetime.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		const columnData = {
+			data: {
+				mode: 'ago',
+			},
+		};
+
+		const wrapper = shallow(<CellDatetime cellData={1518561116} columnData={columnData} />);
+
+		// then
+		expect(wrapper.getElement()).toMatchSnapshot();
```
Comment (acateland) [inline packages/components/src/VirtualizedList/CellDatetime/CellDatetime.test.js]: would prefer a better test checking if the produced value is correct, or to test computeValue wich is exported

guess something wrong is going on https://github.com/Talend/ui/pull/1057/files#diff-b77d66e1616a142ea611a5f02174cdf7R7

File: packages/components/stories/List.js [hunk 2]
```diff
@@ -209,6 +209,84 @@ const props = {
...
+				className: 'item-1-class',
+			},
+			{
+				persistentActions,
+				id: 2,
+				name: 'Super long title to trigger overflow on tile rendering',
+				created: 1518596913333,
+				modified: minusOneHours,
+				author:
+					'Jean-Pierre DUPONT with super super super super super super super super super super super super long name, but there was not enough long text',
```
Comment (talend-glorieux) [inline packages/components/stories/List.js]: Are those long author name relevant to your story?

File: packages/components/stories/List.js [hunk 3]
```diff
@@ -628,4 +719,14 @@ storiesOf('List', module)
 				<List {...props} />
 			</span>
 		</div>
-		));
+	))
+	.add('list with virtualized', () => (
+		<div className="virtualized-list">
+			<h1>List with virtualized and timestamp</h1>
+			<p>CellDatetimeRenderer in action.</p>
+			{/* Do not reproduce!*/}
```
[the remark this answers is not in the document]
Comment (sgendre) [reply packages/components/stories/List.js]: copy/paste from the test above, thanks to notice 👍

File: output/components.eslint.txt [hunk 4]
```diff
@@ -19,11 +19,14 @@ The react/require-extension rule is deprecated. Please use the import/extensions
   1:1  error  '@storybook/react' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
   4:2  error  Unexpected require()                                                                    global-require
 
+/home/travis/build/Talend/ui/packages/components/src/VirtualizedList/CellDatetime/CellDatetime.component.js
+  40:11  error  'rowData' PropType is defined but prop is never used  react/no-unused-prop-types
```
[the remark this answers is not in the document]
Comment (sgendre) [reply output/components.eslint.txt]: ✅

File: packages/components/src/VirtualizedList/CellDatetime/CellDatetime.test.js (hunk 1 above)
Comment (sgendre) [reply packages/components/src/VirtualizedList/CellDatetime/CellDatetime.test.js]: ✅

**Pull request merged**